README.md

```diff
@@ -4,90 +4,90 @@ minideb
+    * Some files that aren't usually required (docs, man pages, locales, caches)
+  * These images also include an `install_packages` command that you can use instead of apt. This takes care of some things for you:
+    * Install the named packages, skipping prompts etc.
+    * Clean up the apt metadata afterwards to keep the image small.
+    * Retrying if apt fails. Sometimes a package will fail to download due to a network issue, and this may fix that, which is particularly useful in an automated build pipeline.
+
+    For example:
+    ```
     $ install_packages apache2 memcached
+    ```
 
-The minideb container image is the base image for many Bitnami-maintained
-language runtimes including 
-[php](https://github.com/bitnami/bitnami-docker-php-fpm), 
-[nodejs](https://github.com/bitnami/bitnami-docker-node), 
-[ruby](https://github.com/bitnami/bitnami-docker-ruby) 
-and infrastructure components
-including [mariadb](https://github.com/bitnami/bitnami-docker-mariadb), 
-[redis](https://github.com/bitnami/bitnami-docker-redis), 
-[nginx](https://github.com/bitnami/bitnami-docker-nginx) and 
-[mongodb](https://github.com/bitnami/bitnami-docker-mongodb). 
+# Adoption of Minideb
+The minideb container image is the base image for many Bitnami-maintained language runtimes including
+  [php](https://github.com/bitnami/bitnami-docker-php-fpm),
+  [nodejs](https://github.com/bitnami/bitnami-docker-node),
+  [ruby](https://github.com/bitnami/bitnami-docker-ruby)
+  and infrastructure components
+  including [mariadb](https://github.com/bitnami/bitnami-docker-mariadb),
+  [redis](https://github.com/bitnami/bitnami-docker-redis),
+  [nginx](https://github.com/bitnami/bitnami-docker-nginx) and
+  [mongodb](https://github.com/bitnami/bitnami-docker-mongodb).
 
-Compatibility
--------------
+# Compatibility
+The image points to the Debian archive, so you are free to install packages from there that you need. However because some `Essential` packages have been removed they may not always install or work correctly.
 
-The image points to the Debian archive, so you are free to
-install packages from there that you need. However because
-some `Essential` packages have been removed they may not
-always install or work correctly.
-
-In those cases you can figure out which package is needed
-and manually specify to install it along with your desired
-packages.
-
-Docker
-------
-
-You can use the image directly, e.g.
+In those cases you can figure out which package is needed and manually specify to install it along with your desired packages.
 
-    $ docker run --rm -it bitnami/minideb:latest
-
-There are tags for the different Debian releases.
-
-    $ docker run --rm -it bitnami/minideb:jessie
+# Building Minideb
+You can build an image yourself if you wish:
+```console
+### Install debootstrap and debian-archive-keyring.
+$ apt-get update && apt-get install -y debootstrap debian-archive-keyring
```

Comment:
Which ubuntu version do you have @sameersbn ? I am on `16.04` and it just works with out those packages. Should we just let user figure out which dependencies need to install `debootstrap` and `debian-archive-keyring` ?